finetuner/__init__.py

```diff
@@ -308,6 +308,64 @@ def get_token() -> str:
     return ft.get_token()
 
 
+def build_model(
+    name: str,
+    model_options: Dict[str, Any] = {},
+    batch_size: int = 32,
+    select_model: Optional[str] = None,
+    device: str = 'cpu',
+    logging_level: str = 'DEBUG',
+    is_onnx: bool = False,
+):
+    """
+    Builds a pre-trained model from a given descriptor.
+
+    :param name: Refers to a pre-trained model, see
+        https://finetuner.jina.ai/walkthrough/choose-backbone/ for a list of all
```

Comment:
I think it is better to refer to the `finetuner.describe_models()` function or mention both. In the documentation we might not display the whole table anymore at some point